src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -4,23 +4,20 @@ cryostat.agent.baseuri=
 
 cryostat.agent.api.writes-enabled=false
 
-# TODO rename these *.ssl.* properties to *.tls.* , and update the README for all these properties
-cryostat.agent.webclient.ssl.trust-all=false
-cryostat.agent.webclient.ssl.verify-hostname=true
+cryostat.agent.webclient.tls.version=TLSv1.2
```

Comment:
Add some documentation to the README for this and the other new properties, please